openapi/webhooks/soft-usage-limit-reached.yaml

```
@@ -0,0 +1,11 @@
+summary: Hard usage limit reached
+operationId: hard-usage-limit-reached
+x-products:
+  - Core
+tags:
+  - Orders
+requestBody:
+  $ref: ../components/requestBodies/Webhooks/UsageLimitWebhook.yaml
+responses:
+  '2xx':
+    description: Returns any 2xx status to indicate that the data received.
```

suggestion: possible typo
```suggestion
    description: Returns any 2xx status to indicate that the is data received.
```
or 
```suggestion
    description: Returns any 2xx status to indicate that data received.
```